code/API_definitions/device-status.yaml

```diff
@@ -131,50 +144,101 @@ paths:
           $ref: "#/components/responses/Generic500"
         503:
           $ref: "#/components/responses/Generic503"
-  /event-subscriptions:
+  /subscriptions:
     post:
       tags:
-        - Device status event subscription
-      summary: 'Create a device status event subscription for a device'
+        - Device status subscription
+      summary: "Create a device status event subscription for a device"
       description: Create a device status event subscription for a device
-      operationId: createDevicStatusEventSubscription
+      operationId: createDeviceStatusSubscription
+      security:
+        - openId:
+            - device-status:roaming-subscriptions:create
```

Comment:
It is a generic operation to create a subscription as the endpoint, description, operationId say, so I think also the scope shouldn't contain "roaming".

Similar for other subscription operations too.

Reply:
@akoshunyadi Yes make sense. Fixed.